f2/src/forms/LocationAnonymousInfoForm.js

```diff
@@ -0,0 +1,78 @@
+/** @jsx jsx */
+import PropTypes from 'prop-types'
+import { jsx } from '@emotion/core'
+import { Trans } from '@lingui/macro'
+import { Form, Field } from 'react-final-form'
+import { NextAndCancelButtons } from '../components/next-and-cancel-buttons'
+import { TextInput } from '../components/TextInput'
+import { Stack, FormControl } from '@chakra-ui/core'
+import { FormLabel } from '../components/FormLabel'
+import { useStateValue } from '../utils/state'
+
+const defaultLocation = {
+  postalCode: '',
+}
+
+export const LocationAnonymousInfoForm = props => {
+  const [data, dispatch] = useStateValue()
+
+  let location
+  if (!data.formData.location) {
+    dispatch({ type: 'saveFormData', data: { location: defaultLocation } })
+    location = defaultLocation
+  } else {
+    location = {
+      ...defaultLocation,
+      ...data.formData.location,
+    }
+  }
+
+  return (
+    <Form
+      initialValues={location}
+      onSubmit={props.onSubmit}
+      render={({ handleSubmit }) => (
+        <Stack as="form" onSubmit={handleSubmit} shouldWrapChildren spacing={6}>
+          <Field name="City">
```

Comment:
Can we use the Field component from our own components? You can checkout our Storybook docs for usage!